test/parameter_test.py

```diff
@@ -688,3 +689,45 @@ class OtherTask(luigi.Task):
         self.assertEqual(MetaTask.saved_value, MetaTask)
         self.assertTrue(luigi.run(['--local-scheduler', '--no-lock'] + 'mynamespace.MetaTask --a other_namespace.OtherTask'.split()))
         self.assertEqual(MetaTask.saved_value, OtherTask)
+
+
+class ForNone(luigi.Task):
+    p = luigi.Parameter(default=None)
+    has_run = False
```

Comment:
If you want to you can use this test-helper: https://github.com/spotify/luigi/blob/master/test/helpers.py#L123